.github/workflows/build-and-test.sh

```diff
@@ -51,5 +49,5 @@ colcon build --packages-up-to buoy_tests --event-handlers console_direct+
 source $COLCON_WS/install/setup.bash
 
 # Test all buoy packages
-colcon test --packages-select-regex=buoy --packages-skip=buoy_msgs --event-handlers console_direct+ --retest-until-pass 10
+colcon test --packages-select-regex=buoy --packages-skip=mbari_wec_utils --event-handlers console_direct+ --retest-until-pass 10
```

Comment:
Package name did not change
```suggestion
colcon test --packages-select-regex=buoy --packages-skip=buoy_msgs --event-handlers console_direct+ --retest-until-pass 10
```